From f42e7879a45d311025914d4a6d243bbe0d068941 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Tue, 28 Oct 2014 18:45:19 +0100
Subject: [PATCH 059/177] profile-resetter: do not tick send-settings by
 default

When the user wishes to reset his preferences, Chromium would suggest
to send over the old settings to Google. I presume that this is for
Google to investigate into the setting combinations that led the user
to issue a rest.

Of course, we cannot let that happen by default. Google is not to get
any information.
---
 chrome/browser/profile_resetter/reset_report_uploader.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/profile_resetter/reset_report_uploader.cc b/chrome/browser/profile_resetter/reset_report_uploader.cc
index ea847b6875..3adcc94a03 100644
--- a/chrome/browser/profile_resetter/reset_report_uploader.cc
+++ b/chrome/browser/profile_resetter/reset_report_uploader.cc
@@ -20,7 +20,7 @@
 
 namespace {
 const char kResetReportUrl[] =
-    "https://sb-ssl.google.com/safebrowsing/clientreport/chrome-reset";
+    "about:blank"; //https://sb-ssl.google.com/safebrowsing/clientreport/chrome-reset";
 
 GURL GetClientReportUrl(const std::string& report_url) {
   GURL url(report_url);
-- 
2.35.1

